client/landing/domains/transfer-away-confirmation/index.jsx

```diff
@@ -32,8 +32,6 @@ class TransferAwayConfirmationPage extends Component {
 	state = {
 		isLoading: false,
 		isProcessingRequest: false,
-		success: false,
-		error: false,
```

Comment:
This `state` assignment is irrelevant because it will be overwritten by the `this.state = this.getLoadingState()` assignment in constructor.

We could remove the constructor and assign the `this.getLoadingState()` return value as instance field here. The `getLoadingState` method should also assign the `isProcessingRequest: false` value. It's not there at this moment.

Reply:
Good spot, addressed in 7c42937dd0219cfcdeb6bed8296c7f183b7e498d